### DISCUSSION TITLE
有没有apisix的技术讨论群

### DISCUSSION THREAD

**Discussion by Lewisyixin** [apache/apisix#11877] [Q&A]
感觉现在github上现在已经没什么活跃度了，问题基本没人回复了，有没有apisix使用者有一些平时讨论问题的技术群，可以分享下嘛

**Comment by darkSheep404** [comment]
> 感觉现在github上现在已经没什么活跃度了，问题基本没人回复了，有没有apisix使用者有一些平时讨论问题的技术群，可以分享下嘛

确实感觉github 活跃度比前两年低了 有官方的QQ群来的我记得,但是讨论问题可能还是github这种形式更方便

**Comment by darkSheep404** [comment]
578997126
